wob.c

```diff
@@ -654,9 +663,9 @@ main(int argc, char **argv)
 					return EXIT_SUCCESS;
 				}
 
-				old_background_color = background_color;
-				old_border_color = border_color;
-				if (fgets_rv == NULL || !wob_parse_input(input_buffer, &percentage, &background_color, &border_color, &bar_color) || percentage > maximum) {
+				old_colors.background = colors.background;
```

Comment:
I think this can be simplified to just:
```suggestion
				old_colors = colors;
```

Reply:
Of course.